ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -258,5 +258,7 @@ app.post('/ehsm', function (req, res) {
   }
 })
 
-app.listen(port, () => {})
+app.listen(port, () => {
+  console.log(`ehsm_ksm_service application listening at http://localhost:${port}`)
```

Comment:
change to
console.log(`ehsm_ksm_service application listening at ${port}...`)

and if there's some one connect to it,  it's better to print out the target ip.

Reply:
it will be fix